common/json.c

```diff
@@ -538,6 +489,15 @@ void json_add_object(struct json_result *result, ...)
 	va_end(ap);
 }
 
+void json_add_escaped_string(struct json_result *result, const char *fieldname,
```

Comment:
Function is defined in `common/json.c`, but is declared in header `common/json_escaped.h`.

Reply:
Oops. I moved it because it wanted access to internal routines in json.c. Will move declaration.